sdk/java/pulumi/src/main/java/com/pulumi/resources/Resource.java

```diff
@@ -376,6 +362,26 @@ private static ImmutableMap<String, ProviderResource> convertToProvidersMap(@Nul
         return result.build();
     }
 
+    private static List<Output<String>> computeAliases(String name, String type, ResourceOptions options) {
```

Comment:
if this is intentionally in this PR then all good, I just looks like a fragment of another PR ;)